deploy/.env.example

```diff
@@ -39,6 +39,16 @@ REQUIRED_NUMBER_OF_VALIDATORS=1
 #If several validators are used, list them separated by space without quotes
 #E.g. VALIDATORS=0x 0x 0x
 VALIDATORS=0x
+#for erc_to_native mode
+#Set to true if RewardableValidators will be used and fee will be charged
+REWARDABLE_VALIDATORS=false
+#If REWARDABLE_VALIDATORS=true, list validators accounts were rewards should be transferred separated by space without quotes
+#E.g. VALIDATORS_REWARD_ACCOUNTS=0x 0x 0x
+VALIDATORS_REWARD_ACCOUNTS=0x
+
+# Fee to be charged for each transfer:
+# E.g. 0.1% fee
+BRIDGE_FEE=0.001
```

Comment:
Consider to have `HOME_TRANSACTIONS_FEE` and `FOREIGN_TRANSACTIONS_FEE`.

Reply:
Done https://github.com/poanetwork/poa-bridge-contracts/pull/134/commits/bdc24f976f515cc1e8560ee531648d048786858c